### PR TITLE
Feature/move to test

### PR DESCRIPTION
**Pull request by lucbel**
Some improvements

### CODE REVIEW

File: src/test/java/com/fullstackmarc/fwkpoc/selenium/pages/google/StackOverflowPage.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.fullstackmarc.fwkpoc.selenium.pages.stackoverflow;
```
Comment (cbmarc) [inline src/test/java/com/fullstackmarc/fwkpoc/selenium/pages/google/StackOverflowPage.java]: Why stackoverflow is in google?

File: src/test/java/com/fullstackmarc/fwkpoc/selenium/pages/google/StackOverflowJobsPage.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.fullstackmarc.fwkpoc.selenium.pages.stackoverflow;
```
Comment (cbmarc) [inline src/test/java/com/fullstackmarc/fwkpoc/selenium/pages/google/StackOverflowJobsPage.java]: Why move stackoverflow to google?

File: src/test/java/com/fullstackmarc/fwkpoc/selenium/pages/google/MyFiddlePage.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.fullstackmarc.fwkpoc.selenium.pages.myfiddle;
```
Comment (cbmarc) [inline src/test/java/com/fullstackmarc/fwkpoc/selenium/pages/google/MyFiddlePage.java]: Why move my fiddle to google?

**Review by cbmarc — COMMENTED** [review]
Please, kindly review my comments

**Comment by lucbel** [comment]
The files have been moved to the proper package.

**Pull request merged**